base/strings/search.jl

```diff
@@ -101,6 +101,8 @@ julia> findfirst("Julia", "JuliaLang")
 findfirst(pattern::AbstractString, string::AbstractString) =
     findnext(pattern, string, firstindex(string))
 
+findfirst(c::AbstractChar, s::AbstractString) = findfirst(equalto(c), s)
```

Comment:
While you're at it, better add a `findnext` method, and change the deprecations to use the new methods where appropriate.